Automattic/woocommerce-payments
# Fix UPE alignment issues on Classic Checkout

Fixes #2609 

#### Changes proposed in this Pull Request

This PR contains minor CSS changes to fix the alignment of UPE Payment Form & Save payment checkbox on Classic checkout.

WC Blocks alignment issues doesn't seem to be present. See [comment](https://github.com/Automattic/woocommerce-payments/issues/2609#issuecomment-1097542722) for reference.

#### Testing instructions

* Enable UPE, add products to cart and go to checkout
* Choose the option `Use a new payment method`
* Notice the difference in padding with the payment form & save payment checkbox ([Screenshot](https://camo.githubusercontent.com/9837ecad543605806b8e2d6f8910571a904dca19d078084ab06227274d6177d5/68747470733a2f2f63646e2d7374642e64726f706c722e6e65742f66696c65732f6163635f313130343230362f686341364362))
* Checkout the branch `fix/upe-alignment` and rebuild assets
* Confirm that the difference in padding is not present. ([Screenshot](https://user-images.githubusercontent.com/1161872/163102459-85d0829e-47aa-4601-9263-340a66f3e487.png))

-------------------


**Post merge**
